app/main.py

```diff
@@ -1,2 +1,48 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.fullness = 0
+        self.buckets = [None] * self.capacity
+
+    def __resizer(self) -> None:
```

Comment:
```suggestion
    def __resize(self) -> None:
```
Name functions using verbs (classes using nouns)